admin/display/menu.php

```diff
@@ -44,6 +48,27 @@ function add_pro_submenu() {
 			$url,
 		);
 	}
+
+	/*
+	 * Opens Upgrade link in menu as new tab.
+	 *
+	 * @since 3.0
+	 */
+	function upgrade_link_aioseop_menu_new_tab() {
+		wp_enqueue_script( 'aioseop_menu_js', AIOSEOP_PLUGIN_URL . 'js/aioseop-menu.js', array( 'jquery' ), AIOSEOP_VERSION, true );
+	}
+
+	/*
+	 * Opens Upgrade link in plugins menu as new tab.
+	 *
+	 * @since 3.0
+	 */
+	function upgrade_link_plugins_menu_new_tab( $hook ) {
+		if ( 'plugins.php' != $hook ) {
```

Comment:
IF checks need to be a strict comparison.

Reply:
Done.